graph_spec.rb

```diff
@@ -52,5 +52,24 @@
 
       expect(g.topological_sort).to eq(expected)
     end
+
+    it 'raises an UnsortableCyclicGraphError when the graph has cycles' do
+      g = Graph.new([
+        [1, 2],
+        [2, 1]
+      ])
+      expect { g.topological_sort }.to raise_error(UnsortableCyclicGraphError)
+    end
+  end
+
+  describe '#cyclic?' do
+    it 'returns true for acyclic graphs' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.